python/pyarrow/types.pxi

```diff
@@ -1122,6 +1150,19 @@ cdef class DurationType(DataType):
         """
         return timeunit_to_string(self.duration_type.unit())
 
+    def to_pandas_dtype(self):
+        """
+        Return the equivalent NumPy / Pandas dtype.
+
+        Examples
+        --------
+        >>> import pyarrow as pa
+        >>> d = pa.duration('ms')
+        >>> d.to_pandas_dtype()
+        timedelta64[ms]
+        """
+        return _get_pandas_type(_Type_TIMESTAMP, self.unit)
```

Comment:
Should this use _Type_DURATION instead? 
(which also raises the question: is this tested? Although I suppose you haven't yet bothered with updating all the tests)

Reply:
Great catch! You're correct that this isn't tested yet.